content/kubernetes/release-notes/k8s-6-2-18-41.md

```diff
@@ -0,0 +1,116 @@
+---
+Title: Redis Enterprise for Kubernetes release notes 6.2.18-41 (Dec 2022)
+linktitle: 6.2.18-41 (Dec 2022)
+description: This is a maintenance release for 6.2.18 and includes bug fixes.
+weight: 60
+alwaysopen: false
+categories: ["Platforms"]
+aliases: [
+    /kubernetes/release-notes/k8s-redis-on-flash.md,
+    /kubernetes/release-notes/k8s-redis-on-flash/, 
+    /kubernetes/release-notes/k8s-6-2-18-41.md,
+    /kubernetes/release-notes/k8s-6-2-18-41/,   ]
+---
+## Overview
+
+The Redis Enterprise K8s 6.2.18-41 is a maintenance release which supports the Redis Enterprise Software release 6.2.18 and contains bug fixes.
+
+The key bug fixes and known limitations are described below.
+
+## Images
+
+This release includes the following container images:
+
+* **Redis Enterprise**: `redislabs/redis:6.2.18-65` or  `redislabs/redis:6.2.18-65.rhel8-openshift` (or `redislabs/redis:6.2.18-65.rhel7-openshift` if upgrading from RHEL 7)
+* **Operator**: `redislabs/operator:6.2.18-41`
+* **Services Rigger**: `redislabs/k8s-controller:6.2.18-41` or `redislabs/services-manager:6.2.18-40` (on the Red Hat registry)
+
+## Bug fixes
+
+* Fixed issues with corrupt image sha (OLM/OpenShift)(RED-88863)
```

Comment:
- Maybe  **corrupt** --> **incorrect** (since it wasn't really corrupt but rather valid image SHA, just not the correct one...)?
- **sha** --> **digest** (or at least, **SHA**, but I think digest is a better term here).
- Add a space before the `(RED-XXXXX)`.